tests/infra/checker.py

```diff
@@ -30,7 +30,8 @@ def __call__(self, rpc_result, result=None, error=None, timeout=2):
                     result, rpc_result.body
                 )
 
-            assert rpc_result.seqno >= 0 and rpc_result.view >= 0, rpc_result
+            if rpc_result.seqno is not None or rpc_result.view is not None:
```

Comment:
Why?

Reply:
Some requests now don't interact with the `Tx` at all, so don't get TxID headers:
https://github.com/microsoft/CCF/pull/2010#discussion_r540426983

Reply:
Sure, but then they should break here, checking they have committed is meaningless.